### PR TITLE
SWAN-11: Add token validation functionality

### PR DESCRIPTION
**Pull request by Arash-Swan**
Add token validation functionality

### CODE REVIEW

File: src/api/token.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+""" Token Related functionalities"""
+
+import requests
+from src.constants.constants import SWAN_API, TOKEN_VALIDATION
+from src.exceptions.exceptions import HTTPError, RequestError
+
+
+def validate_token(token):
```
Comment (pluto-workhub) [inline src/api/token.py:8]: It's better we put this validate part into SDK init prat, the user puts a token into the .env file, and we do validate + get JWT Token when init the sdk

File: src/api/token.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    except requests.exceptions.RequestException as err:
+        # Raise a custom RequestError
+        raise RequestError(f"Request Exception: {err}") from err
+
+    else:
+        # Parsing the JSON response from the server
+        response_data = response.json()
+        response_data["status_code"] = response.status_code
+        return response_data
```
Comment (pluto-workhub) [inline src/api/token.py:57]: Which means this func needs to return JWT Token

File: test/api/test_token.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+""" Token API tests """
+from mock.mock import patch
+from src.api.token import validate_token
+
+
+class TestToken:
+    def test_valid_api_token(self):
```
Comment (pluto-workhub) [inline test/api/test_token.py:7]: If we need change the func then we also need change this

**Review by pluto-workhub — CHANGES_REQUESTED** [review]
Wating for Changes

File: src/api/token.py (hunk 1 above)
Comment (Arash-Swan) [reply src/api/token.py:8]: This API which is responsible for this function does not return the JWT token.
https://github.com/filswan/swan-hub/blob/b600fe9d597a27f636ea1ab1b014e0c5d83fb62b/backend/router/api_token.py#L98

**Pull request merged**